Documentation/project-docs/glossary.md

```diff
@@ -1,82 +1,227 @@
+This refers to the .NET Framework that first shipped in 2002 and has been
+updated on a regular basis since then. It's the main framework folks target
+today and allows you to build a wide variety of applications, such as WinForms,
+WPF, ASP.NET, and command line tools.
 
-The .NET Framework was designed to run on Windows only. Some versions of the .NET Framework come pre-installed with Windows, some require to be installed. However, in both cases the .NET Framework is a system-wide component. Applications do not include .NET Framework DLLs when deploying; the correct .NET version must be on the machine.
+The .NET Framework was designed to run on Windows only. Some versions of the
+.NET Framework come pre-installed with Windows, some require to be installed.
+However, in both cases the .NET Framework is a system-wide component.
+Applications do not include .NET Framework DLLs when deploying; the correct .NET
+version must be on the machine.
 
 ### .NET Core
 
 **Also referred to as**: UWP, ~~Store~~
 
-Originally, .NET Core was the identifier we used to describe the .NET APIs Windows 8 store applications could use. When we designed the API set, we wanted to create a foundation for .NET where portability is a first class concern for the layering and componentization. For more details, read [this blog post](http://blogs.msdn.com/b/dotnet/archive/2014/12/04/introducing-net-core.aspx).
+Originally, .NET Core was the identifier we used to describe the .NET APIs
+Windows 8 store applications could use. When we designed the API set, we wanted
+to create a foundation for .NET where portability is a first class concern for
+the layering and componentization. For more details, read [this blog
+post][introducing-net-core].
 
-Today, .NET Core is no longer just for store applications. .NET Core is the name for the open source, cross-platform stack that ASP.NET Core and UWP applications are built on. The stack includes a set of framework libraries (CoreFX), a JIT based runtime (CoreCLR), an AOT based runtime (CoreRT), and a set of tooling (such as the dotnet CLI).
+Today, .NET Core is no longer just for store applications. .NET Core is the name
+for the open source, cross-platform stack that ASP.NET Core and UWP applications
+are built on. The stack includes a set of framework libraries (CoreFX), a JIT
+based runtime (CoreCLR), an AOT based runtime (CoreRT), and a set of tooling
+(such as the dotnet CLI).
 
-That's why referring to .NET Core as 'Store' is no longer correct. But you can think of today's .NET Core as an evolution of the original APIs available for store applications. Many of the original design goals are still relevant, especially around layering and portability.
+That's why referring to .NET Core as 'Store' is no longer correct. But you can
```

Comment:
I think it is not worth making any connection between .NET Core and "Store" anymore

Reply:
So just remove this paragraph?